cypress/support/generate.user.js

```diff
@@ -0,0 +1,22 @@
+import { faker } from '@faker-js/faker';
+
+function generateWorker () {
+  const randomNumber = Math.random().toString().slice(4);
+  const firstName = faker.person.firstName() + randomNumber;
+  const lastName = faker.person.lastName();
+  const userEmail = faker.internet.email();
+  const age = faker.number.int({ max: 65 });
```

Comment:
The `age` is generated with a maximum value of 65. If there is a minimum age requirement, consider specifying it in the options.